tests/test_rooms/test_analytics_total_durations.py

```diff
@@ -9,52 +12,71 @@
     get_paginated_meetings_total_duration_query_invalid_page,
     get_paginated_meetings_total_duration_invalid_page_result,
     meetings_total_duration_query_for_a_future_date
-    )
+)
+
+with open('events.json', 'r') as outfile:
+    events = json.load(outfile)
 
 
 class TotalDailyDurations(BaseTestCase):
 
-    def test_total_daily_durations(self):
+    @patch("helpers.calendar.analytics_helper.get_google_calendar_events",
```

Comment:
This line of code is repeated multiple times within the file. Is there a way you could limit this?

Reply:
I could put the wrapper in the class level. Let me try this out @Bonifase.

Reply:
This has been handled @Bonifase